README.md

```diff
@@ -35,144 +48,69 @@ Pan domain auth is split into 4 modules.
 
 The `pan-domain-auth-verification` library provides the basic functionality for sigining and verifying login cookies. For
 applications that only need to *VERIFY* an existing login (rather than issue logins themselves) this is the library to use.
-In most cases this will be useful for APIs that are unwilling or unable to offer a user-facing OAuth dance to acquire
-credentials directly, on behalf of the user. Note that this module also includes means for obtaining the public key used
-to do the verification (more details below).
 
-The `pan-domain-auth-core` library provides the core utilities to load the domain settings, create and validate the cookie and
-check if the user has mutlifactor auth turned on (see below). Note this does not include the OAuth dance code or cookie setting
-as these vary based on web framework being used by implementing apps.
+The `pan-domain-auth-core` library provides the core utilities to load settings, create and validate the cookie and
+check if the user has mutli-factor auth turned on when usng Google as the provider.
 
 The `pan-domain-auth-play_2-6` library provide an implementation for play apps. There is an auth action that can be applied to the
 endpoints in your application that will do checking and setting of the cookie and will give you the OAuth authentication mechanism
 and callback. This is the only framework specific implementation currently (due to play being the framework predominantly used at The
 Guardian), this can be used as reference if you need to implement another framework implementation. This library is for applications
 that need to be able to issue and verify logins which is likely to include user-facing applications.
 
-The `pan-domain-auth-example` provides an example app with authentication. This is implemented in play and is used for testing.
```

Comment:
The main change here is to move the `Requirements` section about the `Integrating with your app` section, with a link to skip if you already have everything configured. Users setting up the Grid for the first time found this confusing.

I have also moved code examples to the examples project wherever possible so they are more likely to stay up to date with future changes (because breaking changes would cause compile errors etc).

The README is still a bit spaghetti but I have hopefully removed anything that is no longer true and tidied it up to be a bit more comprehensible